Content.Server/_Goobstation/Heretic/EntitySystems/EldritchItemSystem.cs

```diff
@@ -0,0 +1,67 @@
+using Content.Server.Heretic.Components;
+using Content.Server.Popups;
+using Content.Shared.DoAfter;
+using Content.Shared.Examine;
+using Content.Shared.Heretic;
+using Content.Shared.Interaction.Events;
+
+namespace Content.Server.Heretic.EntitySystems;
+
+public sealed partial class EldritchItemSystem : EntitySystem
```

Comment:
`EldritchItem` is a pretty vague name, especially when this seems to have a very specific function of granting knowledge when used. you could slap an `EldritchItem` component on any heretic item and, without looking at what the system does, i'd probably think it was fine